File: app/src/main/java/restClientService/ServiceUtil.java
package restClientService;

import restClient.RestHandler;
import restClient.RestParms;
import restClient.RestTag;


/**
 * Created by Kelci on 6/18/2018.
 */

public class ServiceUtil {

    public static synchronized void getTasks(RestTag restTag, RestParms restParms, RestHandler restHandler, boolean useCache) {

        new RestGetTasks().call(restTag, restParms, restHandler, useCache);
    }
}

File: app/src/main/java/com/kelci/familynote/model/restService/ServiceUtil.kt
package com.kelci.familynote.model.restService

import restClient.RestHandler
import restClient.RestParms
import restClient.RestTag
import restClientService.RestGetTasks

class ServiceUtil {

    companion object {

        @Synchronized
        fun login(restTag: RestTag, restParms: RestParms, restHandler: RestHandler<*>, useCache: Boolean) {
            RestLogin().call(restTag, restParms, restHandler, useCache)
        }
    }

}